swift/Todo/Controller/AppController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  AppController.swift
+//  Todo
+//
+//  Created by Callum Birks on 15/02/2023.
+//  Copyright © 2023 Couchbase. All rights reserved.
+//
+
+import os
+import SwiftUI
+import CouchbaseLiteSwift
+
+// QE:
+let kQEFeaturesEnabled = true
+
+// Logout Method
+enum LogoutMethod {
+    case closeDatabase, deleteDatabase;
+}
+
+class AppController {
+    public static let logger = Logger()
```

Comment:
Nice, I don't know about the Logger before :)

Reply:
Me neither, it's a nice alternative to using NSLog